src/test/java/uk/gov/hmcts/reform/sscs/services/sftp/SftpChannelAdapterTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.reform.sscs.services.sftp;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.*;
+
+import com.jcraft.jsch.*;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Vector;
+import org.apache.commons.io.IOUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.sscs.config.properties.SftpSshProperties;
+import uk.gov.hmcts.reform.sscs.services.gaps2.files.Gaps2File;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SftpChannelAdapterTest {
+
+    @Mock
+    private JSch jsch;
+
+    @Mock
+    private Session session;
+
+    @Mock
+    private ChannelSftp channel;
+
+    @Mock
+    ChannelSftp.LsEntry entry;
+
+    private SftpSshProperties props = new SftpSshProperties();
+    private String fname1 = "SSCS_Extract_Delta_2000-01-01-01-01-01.xml";
```

Comment:
These variables are only used in one single method. You might want to move then to the method instead.

Reply:
More tests to come.